### PR TITLE
ipi-vpshere-ovn-shared-to-local-gateway-mode-migration

### PR DESCRIPTION
**Pull request by anuragthehatter**

### CODE REVIEW

**Comment by anuragthehatter** [comment]
cc @lihongan @jianlinliu

File: ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/ovn/shared-to-local-gateway-mode-migration/OWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+approvers:
```
Comment (liangxia) [inline ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/ovn/shared-to-local-gateway-mode-migration/OWNERS:1]: Perhaps add yourself to the approver/reviewer list ?

Comment (anuragthehatter) [reply ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/ovn/shared-to-local-gateway-mode-migration/OWNERS:1]: @liangxia Thanks, sure. Just to OWNERS file or I believe both metadatas and OWNERS?

Comment (liangxia) [reply ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/ovn/shared-to-local-gateway-mode-migration/OWNERS:1]: After the OWNERS file is updated, `make update` will render the other files for you.

**Comment by liangxia** [comment]
/hold
for feature to ack

**Comment by anuragthehatter** [comment]
> /hold for feature team to ack

cc @zhaozhanqi

**Pull request merged**